### DISCUSSION TITLE
.SBF parser

### DISCUSSION THREAD

**Discussion by djdan7** [semuconsulting/PyGPSClient#131] [Ideas and Feature Requests]
Hello!
Are you planning to develop an .SBF (Septentrio) protocol parser and add it to your portfolio? It's very well documented, but due to my relatively poor knowledge of binary message processing, I'm having a hard time figuring it out.

It starts from the page 243 in the file below. In need to read mainly three messages: PVTGeodetic, DOP and ReceiverStatus. If you would like to take a look I would be gratefull :)

[mosaic-X5 Firmware v4.14.4 Reference Guide.pdf](https://github.com/semuconsulting/PyGPSClient/files/15143735/mosaic-X5.Firmware.v4.14.4.Reference.Guide.pdf)

**Comment by semuadmin** [comment]
Sorry, no current plans to implement the proprietary SBF protocol, at least not as a voluntary activity.

If someone wants to provide suitable test resources, including a Sepentrio receiver, I may consider it at some point in the future.
